terraform/azure_server/README.md

```diff
@@ -52,6 +52,20 @@ terraform -chdir=terraform/azure_server output
 terraform -chdir=terraform/azure_server destroy -var-file="examples/windows.tfvars"
 ```
 
+## Creating a Windows node with the Containers feature enabled
+
+To trigger the module to create a Windows node with the `Containers` feature enabled, use the example from `windows_containers.tfvars`
```

Comment:
Should we add an explanation of what the `Containers` feature is, maybe with a link to the MS docs, here as well? In terms of why you might want to enable it.

Reply:
I've added an extra sentence to briefly explain what the feature is (basically just enables containerized workloads), but had difficulty finding a MS Docs source that touches on that optional feature specifically. It seems the feature is enabled in the background by Docker or other container runtimes, so doesn't seem to be mentioned explicitly anywhere